Weekly Code Coverage failure upload to coveralls

# Bug Report

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | dev-master              |
| Installed as   | repo                           |

<!-- Please describe your problem here. -->

On Github Action weekly, it got the folllowing errors:

```bash
Coverage:  77.60% (34094/43933)
Collect git info
Read environment variables
Dump submitting json file: /home/runner/work/rector/rector/build/logs/coveralls-upload.json
File size: 6,137.99 kB
Submitting to https://coveralls.io/api/v1/jobs
Client error occurred. status: 422 Unprocessable Entity
Couldn't find a repository matching this job.
elapsed time: 1.502 sec memory: 26.00 MB
Error: Process completed with exit code 1.
```

at https://github.com/rectorphp/rector/runs/1756023886?check_suite_focus=true#step:5:2190

## Minimal PHP Code Causing Issue

Github Action run based on https://github.com/rectorphp/rector/blob/c5132286f8e3f2c01a3fe44986f1d2b8f4f0d9a1/.github/workflows/weekly_code_coverage.yaml

Possible solution: change ${{ secrets.ACCESS_TOKEN }} 

https://github.com/rectorphp/rector/blob/c5132286f8e3f2c01a3fe44986f1d2b8f4f0d9a1/.github/workflows/weekly_code_coverage.yaml#L56

to

${{ secrets.GITHUB_TOKEN }} like in other workflow:

https://github.com/rectorphp/rector/blob/c5132286f8e3f2c01a3fe44986f1d2b8f4f0d9a1/.github/workflows/weekly_pull_requests.yaml#L57

--- a/.github/workflows/weekly_code_coverage.yaml
+++ b/.github/workflows/weekly_code_coverage.yaml
@@ -23,18 +23,9 @@ jobs:
 
                             vendor/bin/phpunit-merger coverage build/logs clover.xml
 
-                            # backup old composer files
-                            cp composer.json composer.json.backup
-                            cp composer.lock composer.lock.backup
-
                             # Coveralls.io
                             composer require --dev php-coveralls/php-coveralls:^2.4
                             vendor/bin/php-coveralls --coverage_clover=clover.xml -v
-
-                            # clean up
-                            rm -rf build
-                            mv composer.json.backup composer.json
-                            mv composer.lock.backup composer.lock
                         branch: 'master'
 
         name: ${{ matrix.actions.name }}

--- a/.github/workflows/weekly_code_coverage.yaml
+++ b/.github/workflows/weekly_code_coverage.yaml
@@ -23,13 +23,18 @@ jobs:
 
                             vendor/bin/phpunit-merger coverage build/logs clover.xml
 
+                            # backup old composer files
+                            cp composer.json composer.json.backup
+                            cp composer.lock composer.lock.backup
+
                             # Coveralls.io
                             composer require --dev php-coveralls/php-coveralls:^2.4
                             vendor/bin/php-coveralls --coverage_clover=clover.xml -v
 
                             # clean up
-                            git checkout .
-                            git clean -xfd build
+                            rm -rf build
+                            mv composer.json.backup composer.json
+                            mv composer.lock.backup composer.lock
                         branch: 'master'
 
         name: ${{ matrix.actions.name }}
@@ -53,4 +58,4 @@ jobs:
             -
                 run: ${{ matrix.actions.run }}
                 env:
-                    COVERALLS_REPO_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+                    COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}

